MeshLib/PropertyVector.h

```diff
@@ -72,11 +72,11 @@ friend class Properties;
 	/// @param mesh_item_type the values of the property are either assigned to
 	/// nodes or cells (see enumeration MeshItemType) (default:
 	/// MeshItemType::Cell)
-	/// @param tuple_size the number of elements of a tuple (default: 1)
+	/// @param n_components the number of elements of a tuple (default: 1)
 	explicit PropertyVector(std::string const& property_name,
 		MeshItemType mesh_item_type = MeshItemType::Cell,
-		std::size_t tuple_size = 1)
-		: std::vector<PROP_VAL_TYPE>(), _tuple_size(tuple_size),
+		std::size_t n_components = 1)
```

Comment:
Actually the default arguments were not used anyway, so removed them in this PR. The `operator[]` needs discussion.